### ISSUE TITLE
About the evaluation for Full Surround Monodepth from Multiple Cameras

### ISSUE THREAD

**Issue by weiyithu** [TRI-ML/packnet-sfm#210]
<img width="239" alt="微信图片_20220127120640" src="https://user-images.githubusercontent.com/32617425/151290197-4abf1011-80ad-40a9-8e59-4bf9830c77b7.png">
Thanks for your great work. I have some questions about the evaluation on DDAD. For the validation set, DDAD has 3950 samples, we should get 3950*6=23700 images but not 15800 images. Also for the training set, we should get 12650*6=75900 images but not 63250 images.

**Comment by hjxwhy** [comment]
I use the DGP data load to load the DDAD six camera at the same time, tqdm show it has 12k, I guess it is because the DGP will discard some small action frames or static frames, the DGP code is so complicated, so I have not read carefully, I just use it.

**Comment by weiyithu** [comment]
Thanks for your responce! I think 12k (12650) is right but total number of images should not be 63250. By the way, have you successfully reproduced the results ?

**Comment by VitorGuizilini-TRI** [comment]
I apologize if there is any confusion. We are preparing an updated version of the paper and will upload it to arXiv soon, and we have started looking into open-sourcing the FSM code.

**Comment by weiyithu** [comment]
Hi author, do you use the depth from DDAD dataloader with original resolution (1216 x 1936) for evaluation?

**Comment by hjxwhy** [comment]
> I apologize if there is any confusion. We are preparing an updated version of the paper and will upload it to arXiv soon, and we have started looking into open-sourcing the FSM code.

No, I have also not reproduced the result. I have tried many setups but its RMSE is still worse than the paper 5, so I give up now...

**Comment by hhhharold** [comment]
> I apologize if there is any confusion. We are preparing an updated version of the paper and will upload it to arXiv soon, and we have started looking into open-sourcing the FSM code.

Hello, I am confused that do you using automask when calculation photometric loss on FSM? The description of that on the paper is not clear.

**Comment by weiyithu** [comment]
> > I apologize if there is any confusion. We are preparing an updated version of the paper and will upload it to arXiv soon, and we have started looking into open-sourcing the FSM code.
> 
> Hello, I am confused that do you using automask when calculation photometric loss on FSM? The description of that on the paper is not clear.

Welcome to focus on our work https://arxiv.org/abs/2204.03636 and we provide the self-occlusion mask of DDAD.

**Comment by weiyithu** [comment]
> > I apologize if there is any confusion. We are preparing an updated version of the paper and will upload it to arXiv soon, and we have started looking into open-sourcing the FSM code.
> 
> Hello, I am confused that do you using automask when calculation photometric loss on FSM? The description of that on the paper is not clear.

Yeah, we annotate and use self-occlusion mask in DDAD.
